src/TheThingsNetwork.cpp

```diff
@@ -7,9 +7,12 @@
 #define debugPrintLn(...) { if (debugStream) debugStream->println(__VA_ARGS__); }
 #define debugPrint(...) { if (debugStream) debugStream->print(__VA_ARGS__); }
 
-void TheThingsNetwork::init(Stream& modemStream, Stream& debugStream) {
-  this->modemStream = &modemStream;
-  this->debugStream = &debugStream;
+void TheThingsNetwork::init(Stream& modemStream) {
+  this->modemStream = &modemStream; 
+  if (!this->debugStream) {
+    Serial.begin(9600);
+    TheThingsNetwork::setDebugSerial(Serial);
+  }
```

Comment:
The [Arduino Reference](https://www.arduino.cc/en/Reference/Serial) does seem to suggest we can assume `Serial` is there:

> All Arduino boards have at least one serial port (also known as a UART or USART): `Serial`. It communicates on digital pins 0 (RX) and 1 (TX) as well as with the computer via USB.

According to the [IfSerial Reference](https://www.arduino.cc/en/Serial/IfSerial) on Leonardo Boards it `Serial` will only be true if it's open, but indeed on others it will always be `true`.

All in all... it looks like there's too many uncertainties and we might be better off just keeping init as it is and have no `set*Serial()`.

@johanstokking let's close then?